packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -656,6 +668,18 @@ export interface ScmExt {
     $setSourceControlSelection(sourceControlHandle: number, selected: boolean): Promise<void>;
 }
 
+export namespace TimelineCommandArg {
+    export function is(arg: Object | undefined): arg is TimelineCommandArg {
+        return !!arg && typeof arg === 'object' && 'timelineHandle' in arg;
+    }
+}
+export interface TimelineCommandArg {
+    timelineHandle: string;
+    source: string;
+    uri: string;
+    isSingleUri: boolean;
```

Comment:
could you elaborate what does it mean?

Reply:
Now passing the `uri` argument directly, removed this flag.